<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>476</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>CasparCG MAV Client</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/Images/CasparCG.ico</normaloff>:/Images/CasparCG.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Main</string>
         </attribute>
         <widget class="QGroupBox" name="groupBox">
          <property name="geometry">
           <rect>
            <x>710</x>
            <y>260</y>
            <width>261</width>
            <height>141</height>
           </rect>
          </property>
          <property name="title">
           <string>CasparCG Server</string>
          </property>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>221</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Address:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
          <widget class="QPushButton" name="btnDisconnect">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>122</x>
             <y>100</y>
             <width>111</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Disconnect</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="edtHost">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>40</y>
             <width>221</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>127.0.0.1:5250</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="chkAutoConnect">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>70</y>
             <width>221</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Connect automatically</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btnConnect">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>100</y>
             <width>111</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>580</x>
            <y>10</y>
            <width>111</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Current frame</string>
          </property>
         </widget>
         <widget class="QTableWidget" name="tblReplays">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>431</width>
            <height>192</height>
           </rect>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <column>
           <property name="text">
            <string>Opis</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Begin</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>End</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Length</string>
           </property>
          </column>
         </widget>
         <widget class="QPushButton" name="btnBegin">
          <property name="geometry">
           <rect>
            <x>450</x>
            <y>10</y>
            <width>111</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string>New</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnEnd">
          <property name="geometry">
           <rect>
            <x>450</x>
            <y>110</y>
            <width>111</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string>Set end</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnSpeed1">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>320</y>
            <width>71</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnSpeed07">
          <property name="geometry">
           <rect>
            <x>180</x>
            <y>320</y>
            <width>41</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string>0.7</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnSpeed05">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>320</y>
            <width>61</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string>0.5</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnSpeed03">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>320</y>
            <width>41</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string>0.3</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnSpeedN05">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>320</y>
            <width>51</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string>-0.5</string>
          </property>
         </widget>
         <widget class="QLabel" name="lblReplayStatus">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>380</y>
            <width>691</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Replay: stopped</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnSpeed0">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>320</y>
            <width>51</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
         <widget class="QLCDNumber" name="lcdFrame">
          <property name="geometry">
           <rect>
            <x>580</x>
            <y>30</y>
            <width>111</width>
            <height>31</height>
           </rect>
          </property>
          <property name="digitCount">
           <number>10</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
         <widget class="QPushButton" name="btnStop">
          <property name="geometry">
           <rect>
            <x>370</x>
            <y>320</y>
            <width>71</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string>STOP</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnDelete">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>450</x>
            <y>170</y>
            <width>111</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnPlayAdd">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>230</y>
            <width>51</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
         <widget class="QSpinBox" name="spnPlayStart">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>230</y>
            <width>91</width>
            <height>41</height>
           </rect>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>1576800000</number>
          </property>
         </widget>
         <widget class="QSpinBox" name="spnPlayEnd">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>230</y>
            <width>91</width>
            <height>41</height>
           </rect>
          </property>
          <property name="maximum">
           <number>1576800000</number>
          </property>
         </widget>
         <widget class="QLabel" name="lblPlayFrames">
          <property name="geometry">
           <rect>
            <x>330</x>
            <y>230</y>
            <width>111</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Frames: 0</string>
          </property>
         </widget>
         <widget class="QLabel" name="lblPlayLength">
          <property name="geometry">
           <rect>
            <x>330</x>
            <y>250</y>
            <width>111</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Length: 0s</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnPlayUpdate">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>270</x>
            <y>230</y>
            <width>51</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_11">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>210</y>
            <width>61</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Play from:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_12">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>210</y>
            <width>61</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Play to:</string>
          </property>
         </widget>
         <widget class="QLabel" name="lblRecordFps">
          <property name="geometry">
           <rect>
            <x>586</x>
            <y>60</y>
            <width>101</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>0fps/0fps</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QCheckBox" name="chkStopAtEnd">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>280</y>
            <width>161</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Stop playback at end</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="btnUpdateBegin">
          <property name="geometry">
           <rect>
            <x>450</x>
            <y>70</y>
            <width>111</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Set begin</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="chkUpdateStart">
          <property name="geometry">
           <rect>
            <x>180</x>
            <y>280</y>
            <width>191</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Update start and end</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QProgressBar" name="proPlayback">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>303</y>
            <width>431</width>
            <height>16</height>
           </rect>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>1</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="textVisible">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="geometry">
           <rect>
            <x>710</x>
            <y>10</y>
            <width>261</width>
            <height>241</height>
           </rect>
          </property>
          <property name="title">
           <string>Config</string>
          </property>
          <widget class="QLineEdit" name="edtLayer">
           <property name="geometry">
            <rect>
             <x>220</x>
             <y>40</y>
             <width>31</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>25</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="edtFile">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>40</y>
             <width>161</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>replay1</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>251</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>replay file / play channel / play layer</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="edtChannel">
           <property name="geometry">
            <rect>
             <x>180</x>
             <y>40</y>
             <width>31</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>1</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btnStop2">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>120</y>
             <width>121</width>
             <height>21</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>STOP [ch]</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btnRemove2">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>140</y>
             <width>121</width>
             <height>21</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>REMOVE [ch] REPLAY</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btnAdd2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>140</y>
             <width>121</width>
             <height>21</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>ADD [ch] REPLAY [file]</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btnPlay2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>120</y>
             <width>121</width>
             <height>21</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>PLAY [ch] [device]</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btnDiag">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>160</y>
             <width>241</width>
             <height>21</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>DIAG</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="edtDevice">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>90</y>
             <width>161</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>DECKLINK 1</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_13">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>70</y>
             <width>251</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>replay device / record channel</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="edtRecordChannel">
           <property name="geometry">
            <rect>
             <x>180</x>
             <y>90</y>
             <width>31</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>2</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btnSaveAs">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>190</y>
             <width>81</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Save as...</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btnSave">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>190</y>
             <width>61</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btnLoad">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>190</y>
             <width>81</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Load...</string>
           </property>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>Log</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QPlainTextEdit" name="edtLog">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>About</string>
         </attribute>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>301</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>(c) 2015 Krzysztof Pyrkosz &lt;pyrkosz@o2.pl&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_14">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>951</width>
            <height>141</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>CasparCG MAV Client is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.

CasparCG MAV Client is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.

You should have received a copy of the GNU General Public License along with CasparCG MAV Client. If not, see http://www.gnu.org/licenses/.</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_15">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>301</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>CasparCG MAV Client v. 0.1</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
